src/OrchardCore/OrchardCore.Setup.Core/SetupService.cs

```diff
@@ -161,6 +161,11 @@ private async Task<string> SetupInternalAsync(SetupContext context)
                 shellSettings["Schema"] = context.Properties.TryGetValue(SetupConstants.DatabaseSchema, out var schema) ? schema?.ToString() : null;
             }
 
+            if (shellSettings["DatabaseProvider"] == DatabaseProviderValue.Sqlite && string.IsNullOrEmpty(shellSettings["DatabaseName"]))
+            {
+                shellSettings["DatabaseName"] = context.Properties.TryGetValue(SetupConstants.DatabaseName, out var dbName) ? dbName?.ToString() : "orchardcore.db";
```

Comment:
Perhaps,

```suggestion
                shellSettings["DatabaseName"] = context.Properties.TryGetValue(SetupConstants.DatabaseName, out var dbName) ? dbName?.ToString() : "OrchardCore.db";
```

Reply:
Sebastien Requested it lower case so it is normalized file name.

Reply:
Normalized with what, you mean?